packages/core/package.json

```diff
@@ -49,6 +49,8 @@
   },
   "dependencies": {
     "@ethersproject/providers": "^5.5.1",
+    "@gnosis.pm/safe-apps-provider": "^0.11.0",
+    "@gnosis.pm/safe-apps-sdk": "^7.3.0",
```

Comment:
I think new providers should be independent packages so we dont have third party dependencies for the core of this lib. 

If we allow gnosis, we should allow 100 more

Reply:
sure. the possible solution could be expose needed connector api to allow third party to integrate they own, but this is quite lacking right now.

or set these pkgs as peer deps.

Reply:
I was going through this, and I might have a workaround for this. Users won't probably even need to many connectors, but since this is a monorepo, what if we published other connectors under `@wagmi/connector-<service>`, such as `@wagmi/connector-gnosis` or something as such.

That would be totally opt-in and you install what you need.